package.json

```diff
@@ -29,15 +29,18 @@
     "node": ">=10.0.0"
   },
   "dependencies": {
-    "postcss": "^7.0.27",
-    "postcss-values-parser": "^3.2.0"
+    "postcss-values-parser": "^4.0.0"
+  },
+  "peerDependencies": {
+    "postcss": "^8.0.0"
   },
   "devDependencies": {
     "@babel/core": "^7.9.0",
     "@babel/preset-env": "^7.9.5",
     "babel-eslint": "^10.1.0",
```

Comment:
Unrelated but if you would like to do a bit clean up before a major bump, I suggest replacing `babel-eslint` by `@babel/eslint-parser`.